doc/platforms.md

```diff
@@ -30,4 +24,4 @@ To build the game for production, using skia, use the following:
 
 Support on web, even on the Flame side is still experimental, and some methods may not work, for example `Flame.util.setOrientation` or `Flame.util.fullScreen` don't work on web, they can be called, but nothing will happen.
 
-Another example, pre caching audio using `Flame.audio` also don't work due to Audioplayers not supporting it on web. This can ben workarounded by using the `http` package, and requesting a get to the audiofile, that will make the browser cache the file prodcing the same effect seem on mobile.
+Another example, pre caching audio using `flame_audio` package, also don't work due to Audioplayers not supporting it on web. This can ben workarounded by using the `http` package, and requesting a get to the audiofile, that will make the browser cache the file prodcing the same effect seem on mobile.
```

Comment:
Grammar/Spelling:

Another example, pre caching audio using `flame_audio` package, also doesn't work due to Audioplayers not supporting it on web. This can be worked around by using the `http` package, and requesting a get to the audiofile, that will make the browser cache the file producing the same effect seem on mobile.

Reply:
```
Another example: pre caching audio using flame_audio package also doesn't work...
```

requesting a get => requesting a `GET`
audiofile => audio file

```
audio file; that will make the browser
```